src/styles/blocks/main.scss

```diff
@@ -0,0 +1,21 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  font-family: $main-font-famaly;
```

Comment:
There is a typo in the variable name '$main-font-famaly'. It should be '$main-font-family'. Ensure that the variable is correctly defined and imported.